config/locales/en/editions/editions.yml

```diff
@@ -0,0 +1,4 @@
+en:
+  editions:
+    non_manage_live_history_mode_permission:
```

Comment:
If you use "permission" in the name here you should also use it in the view file names for consistency.

Reply:
This no longer applies as per using one template